### ISSUE TITLE
Conditional Args for Mutation

### ISSUE THREAD

**Issue by xpromx** [Folkloreatelier/laravel-graphql#251]
Hello,

I'm trying to create a mutation that can expect and user id or email to perform the update,
there is a way to do that from the args rules?

id  || email = required

or i have to write that logic inside the resolve method?

Thanks

**Comment by faiverson** [comment]
No, there isn't an OR in the args. They are not rules, they are all the possible parameters. You need to declare both in args, probably with all other attributes you need to change.
Then, in the resolver you can search by email or id.
You have a rules function too, in case you need to make some validations like you do in a form
